pkgs/applications/office/beamerpresenter/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, mkDerivation, fetchFromGitHub, makeDesktopItem,
+  qmake, qtbase, poppler, qtmultimedia }:
+
+mkDerivation rec {
+  pname = "beamerpresenter";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "stiglers-eponym";
+    repo = "BeamerPresenter";
+    rev = "v${version}";
+    sha256 = "0j7wx3qqwhda33ig2464bi0j0a473y5p7ndy5f7f8x9cqdal1d01";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ qmake ];
+  buildInputs = [ qtbase qtmultimedia poppler ];
+
+  postPatch = ''
+    # Fix location of poppler-*.h
+    shopt -s globstar
+    for f in **/*.{h,cpp}; do
+      substituteInPlace $f --replace '#include <poppler-' '#include <poppler/qt5/poppler-'
+    done
+  '';
+
+  installPhase = ''
+    install -m755 beamerpresenter -Dt $out/bin/
+    install -m644 man/beamerpresenter.1 -Dt $out/share/man/man1/
+    install -m644 man/beamerpresenter.conf.5 -Dt $out/share/man/man5/
```

Comment:
Use `installManPage` from `installShellFiles`.

Reply:
Replaced and amended.